annotation-client/src/test/resources/subtitle/minimal_qt_right.json

```diff
@@ -4,7 +4,8 @@
     "type": "FullTextResource",
     "language": "it",
     "value": "{QTtext}{timescale:1000}{font:Arial}{size:16}{backColor:0,0,0}{textColor:65535,65535,65535}{width:420}{justify:center}{dropShadow:on}{dropShadowOffset:1,1}{dropShadowTransparency:200}\n[00:00:01.250]\n[Dotsub Any Video Any Language]",
-    "format": "video/quicktime"
+    "format": "video/quicktime",
+    "edmRights": "http://creativecommons.org/licenses/by-sa/1.0/"
```

Comment:
The [template files for the swagger console](http://annotation-api-test.eanadev.org/jsp/template/jsonld.jsp) need to be updated to include the captions and subtitles

Reply:
Improved.